### PR TITLE
Nexus Integrated with orbit

### PR DESCRIPTION
**Pull request by web3cook**
**Description**

1. Integrated Nitro contracts with nexus contract
2. Added script to update nitro contract
3. Added holesky as one of the networks

### CODE REVIEW

File: scripts/deployContractsRollup.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  contractAddress: string,
+  constructorArguments: any
+) {
+  await run("verify:verify", {
+    address: contractAddress,
+    constructorArguments: constructorArguments,
+  });
+}
+async function main() {
+  const ExecutorContract = "0xda8636b0d4f31e87d6e7abe1eaa7a17caa9bc487";
```
Comment (umershaikh123) [inline scripts/deployContractsRollup.ts:16]: @RohitAudit what is the function of Executor contract ?

File: hardhat.config.ts [hunk 2]
```diff
@@ -175,6 +184,14 @@ module.exports = {
           browserURL: 'https://sepolia-explorer.arbitrum.io/',
         },
       },
+      {
+        network: 'holesky',
+        chainId: 17000,
+        urls: {
+          apiURL: "https://api-holesky.etherscan.io/api",
```
Comment (umershaikh123) [inline hardhat.config.ts:191]: @RohitAudit 
is this our holesky server endpoint for apiURL?

File: src/nexus/NexusBridgeDAO.sol [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+/**
+ * @title Nexus Bridge DAO Contract
+ * @author RohitAudit
+ * @dev This contract enables DAO governing body of the rollup to claim the staking rewards
+ * from the bridge contract and use it for Ecosystem incentives etc.
+ */
+abstract contract NexusBridgeDAO is NexusBaseBridge {
+    uint256 public rewardsClaimed;
+    // to be changed rollup DAO
+    address public constant DAO = 0x14630e0428B9BbA12896402257fa09035f9F7447;
```
Comment (umershaikh123) [inline src/nexus/NexusBridgeDAO.sol]: @RohitAudit 
is this a multi sig EOA address or Contract address?

File: scripts/deployContractsRollup.ts (hunk 1 above)
Comment (web3cook) [reply scripts/deployContractsRollup.ts:16]: executor contract is used to update any params or contract implementation in the nitro contracts

File: hardhat.config.ts (hunk 2 above)
Comment (web3cook) [reply hardhat.config.ts:191]: this is for verification of contracts on etherscan

File: src/nexus/NexusBridgeDAO.sol (hunk 3 above)
Comment (web3cook) [reply src/nexus/NexusBridgeDAO.sol]: This can be any address. For now it is EOA address

**Review by umershaikh123 — APPROVED** [review]
approved

**Review by rishav-eulb — APPROVED** [review]

**Pull request merged**